src/virtualenv/seed/via_app_data/pip_install/base.py

```diff
@@ -130,13 +130,13 @@ def _console_scripts(self):
 
     def _create_console_entry_point(self, name, value, to_folder, version_info):
         result = []
-        from distlib.scripts import ScriptMaker
+        from distlib.scripts import _enquote_executable, ScriptMaker
```

Comment:
Is this not available publicly, bit nervous to really on a private method 🤔 did you open an issue with distutils that maybe this should happen automatically?

Reply:
created https://bitbucket.org/pypa/distlib/issues/135/expose-_enquote_executable-as-public

Reply:
Thanks for opening the issue!